openssl-dynamic/src/main/c/jnilib.c

```diff
@@ -110,6 +114,16 @@ JNIEXPORT jint JNICALL JNI_OnLoad_netty_tcnative(JavaVM *vm, void *reserved)
     TCN_GET_FIELD(env, keyMaterialClass, keyMaterialPrivateKeyFieldId,
                    "privateKey", "J", JNI_ERR);
 
+//
+// BoringSSL doesn't have OCSP support as far as I know.
+//   https://www.imperialviolet.org/2015/10/17/boringssl.html
+//
+#ifndef OPENSSL_IS_BORINGSSL
+    if (io_netty_tcnative_jni_ocsp_Native_OnLoad(env) != JNI_OK) {
```

Comment:
can we just group the code into `ssl.c` and `sslcontext.c` instead of introducing new modules? That would be more consistent and helps orient you as to what the scope the methods apply to.